modules/gitops-release-notes-1-17-0.adoc

```diff
@@ -0,0 +1,154 @@
+
+[id="fixed-issues-1-17-0_{context}"]
+== Fixed issues
+
+[id="gitops-1-17-0-fixed-rolebinding-errors-by-long-SourceNamespace-project-names_{context}"]
+=== Fixed `RoleBinding` errors caused by long `SourceNamespace` project names
+
+* Before this update, the Argo CD Operator failed to create `RoleBindings` when `sourceNamespaces` contained long namespace names. This issue was caused by Kubernetes label validation errors, as namespace names exceeding `63` characters exceeded the label value length limit. The Argo CD Operator attempted to use the full namespace name as a label value, resulting in reconciliation failures with errors such as `must be no more than 63 characters`. With this update, the issue has been fixed by introducing a `truncateWithHash` function that:
+
+** Truncates names to comply with the 63-character limit
+** Appends a deterministic hash suffix to maintain uniqueness
+** Ensures consistent and unique label values for long namespace names
++
+As a result, `RoleBindings` are now created successfully for long `sourceNamespaces`. link:https://issues.redhat.com/browse/GITOPS-7018[GITOPS-7018]
+
+[id="gitops-1-17-0-fine-grained-rbac-inheritance-enabled-by-default_{context}"]
+=== Fine-grained RBAC inheritance enabled by default
+
+* Before this update, fine-grained RBAC inheritance was disabled in Argo CD 3.0, restricting update and delete actions to the Application resources. This prevented users from modifying sub-resources without defining additional policies. With this update, fine-grained RBAC inheritance is enabled in the Argo CD config map by default. Users can control access to Application sub-resources using inherited RBAC rules in newer Argo CD versions. link:https://issues.redhat.com/browse/GITOPS-6889[GITOPS-6889]
+
+[id="gitops-1-17-0-fixed-replica-count-for-redis-ha-haproxy-to-ensure-ha_{context}"]
+=== Fixed replica count for `redis-ha-haproxy` to ensure high availability
+
+* Before this update, a misconfiguration caused the `redis-ha-haproxy` deployment to use only one replica, resulting in intermittent access issues with the Redis cache. With this update, the number of `redis-ha-haproxy` replicas now matches the High Availability (HA) configuration. Two replicas are created by default, improving Redis cache availability and resilience. link:https://issues.redhat.com/browse/GITOPS-6806[GITOPS-6806]
+
+[id="gitops-1-17-0-gitops-plugin-pods-comply-with-pod-security-policy_{context}"]
+=== `gitops-plugin` Pods comply with Pod Security policy
+
+* Before this update, `gitops-plugin` Pods failed to start due to missing security context properties, resulting in non-compliance with the Pod security policy. This caused the {gitops-title} Operator to fail when creating these Pods. With this update, `gitops-plugin` includes the required security context settings, ensuring compliance with the Pod security policy. link:https://issues.redhat.com/browse/GITOPS-6777[GITOPS-6777]
```

Comment:
The plugin pod failed to start instead of the GitOps operator failed, right @chetan-rns ?

Reply:
Fixed.